test/integration/test/custom_vocabularies.test.js

```diff
@@ -29,3 +29,17 @@ test('Can retreive submitted custom vocabulary', async (done) => {
         })()
     }, 15000);
 }, 300000);
+
+test('Can delete submitted custom vocabulary', async (done) => {
+    const informationSubmit = await client.submitCustomVocabularies([{phrases:['some','custom','vocabularies']}]);
+
+    var intervalObject = setInterval(function(){
```

Comment:
The integration tests fails because the custom vocabulary did not finish processing. You'll have to wait for the cv to either succeed or fail to delete it.